src/SignUpIn/DelivererGroup/DelivererGroupSignUp1.js

```diff
@@ -0,0 +1,104 @@
+            <option value="Maine">Maine</option>
+            <option value="Maryland">Maryland</option>
+            <option value="Massachusetts">Massachusetts</option>
+            <option value="Michigan">Michigan</option>
+            <option value="Minnesota">Minnesota</option>
+            <option value="Mississippi">Mississippi</option>
+            <option value="Missouri">Missouri</option>
+            <option value="Montana">Montana</option>
+            <option value="Nebraska">Nebraska</option>
+            <option value="Nevada">Nevada</option>
+            <option value="New Hampshire">New Hampshire</option>
+            <option value="New Jersey">New Jersey</option>
+            <option value="New Mexico">New Mexico</option>
+            <option value="New York">New York</option>
+            <option value="North Carolina">North Carolina</option>
+            <option value="Ohio">Ohio</option>
+            <option value="Oklahoma">Oklahoma</option>
+            <option value="Oregon">Oregon</option>
+            <option value="Pennsylvania">Pennsylvania</option>
+            <option value="Rhode Island">Rhode Island</option>
+            <option value="South Carolina">South Carolina</option>
+            <option value="South Dakota">South Dakota</option>
+            <option value="Tennessee">Tennessee</option>
+            <option value="Texas">Texas</option>
+            <option value="Utah">Utah</option>
+            <option value="Vermont">Vermont</option>
+            <option value="Virginia">Virginia</option>
+            <option value="Washington">Washington</option>
+            <option value="West Virginia">West Virginia</option>
+            <option value="Wisconsin">Wisconsin</option>
+            <option value="Wyoming">Wyoming</option>
+            </select>
+            <input ref="zip" type="text" id="zip" className="form-component" placeholder="Zip Code" defaultValue={this.props.fieldValues.zip}/>
+            </span>
+        </div>
+
+        <div className="disclaimer">
+            All fields are required unless specified
+            </div>
+        <div className="buttons">
+            <span className="cancel" onClick={this.props.previousStep} >CANCEL</span>
+            <span className="next" onClick={this.props.nextStep}>NEXT</span>
+        </div>
+    </div>
+        )
+    }
+    nextStep(e) {
+        e.preventDefault()
+        console.log(this);
+        console.log(this.refs);
+        // Get values via this.refs
+        var data = {
+            organizationName: ReactDOM.findDOMNode(this.refs.organizationName).value,
+            numVolunteers: ReactDOM.findDOMNode(this.refs.numVolunteers).value,
+            address1: ReactDOM.findDOMNode(this.refs.address1).value,
+            address2: ReactDOM.findDOMNode(this.refs.address2).value,
+            city: ReactDOM.findDOMNode(this.refs.city).value,
+            state: ReactDOM.findDOMNode(this.refs.state).value,
```

Comment:
You should also be able to do `e.target` to get the values. Eg, `e.target.organizationName`, `e.target.numVolunteers`, etc. Correct me if I'm wrong, but it's probably more efficient than calling `ReactDOM.findDOMNode()` since you already have it in the `event` passed into this function.

Reply:
I tried switching it to that and it didn't work. e is referencing the "next" button so I think it couldn't find the other fields outside of it.